### DISCUSSION TITLE
fields missing in notice-subtype form definitions

### DISCUSSION THREAD

**Discussion by klmdb** [OP-TED/eForms-SDK#707] [Q&A] [answered]
The `TransmissionDate` & `TransmissionTime` fields are shown in the notice examples, and they are defined in the conceptual model, BUT they are not included into any of the notice subtype form definitions:

```
<!-- examples/notices/can_24_maximal.xml -->

	<ext:UBLExtensions>
		<ext:UBLExtension>
			<ext:ExtensionContent>
				<efext:EformsExtension>

					<!-- "id" : "BT-803(d)-notice", "name" : "Notice Dispatch Date eSender (date)"-->
					<efbc:TransmissionDate>2023-04-23+01:00</efbc:TransmissionDate>
					<!-- "id" : "BT-803(t)-notice", "name" : "Notice Dispatch Date eSender (time)"-->
					<efbc:TransmissionTime>21:00:00+01:00</efbc:TransmissionTime>
```

field ids:
- `BT-803(d)-notice`
- `BT-803(t)-notice`

**Comment by rouschr** [comment]
Ok, we will look into this for the visual model / notice subtype definitions (notice-types), possibly do something about this in 1.9

**Comment by rouschr** [comment, marked as answer]
According to internal discussions, this specific date and time should not be set via the forms in the user interface, but it should be set securely (back-end) by your system before sending a notice.
